packages/react-components/src/components/ParticipantList.tsx

```diff
@@ -74,7 +81,7 @@ const onRenderParticipantsDefault = (
         <ParticipantItem
           key={index}
```

Comment:
Tangential to this PR, but an important bug: list indices should not be used as keys as it can lead to correctness problems. We should use `participant.userId` instead.